### DISCUSSION TITLE
Does open meteo provide data from India and other Asian countries

### DISCUSSION THREAD

**Discussion by albin-baby-2002** [open-meteo/open-meteo#334] [Polls]
looking for a quick response.

nb: first time using github discussion don't know the etiquette of this platform . Advance sorry for any mistakes

**Comment by patrick-zippenfenig** [comment]
Hi,
thanks for asking. India and most parts of Asia are only covered by global weather models. With resolutions of ~10 km, global models do not represent local effect well (winds in complex mountainous terrain, showers / convective precipitation or urban heat islands), however global models forecast upcoming weather changes quite well.

If there are any weather models available as gridded data (e.g. GRIB or NetCDF files) I would happily integrate those forecast and make them easily accessible through the weather API
